### DISCUSSION TITLE
0167 Navigation is weird on iPad

### DISCUSSION THREAD

**Discussion by surajbarkale** [pointfreeco/episode-code-samples#101] [Q&A]
Here is a screen recording of the code running on iPad (it is slightly better in landscape mode):

https://user-images.githubusercontent.com/1241310/141021712-0b4a1fac-4b4e-484c-afe7-b54d61182512.mp4

A real app will require separate container views for iPad and iPhone. I am not sure if the leaf views need to be tweaked as well. Please plan an episode/blog post focusing on addressing this pain point.

Thanks!

**Comment by stephencelis** [comment]
Hi @surajbarkale! For the purpose of the episode we only got things to behave in landscape mode. Supporting more advanced iPad navigation takes a bit more work than we could squeeze into the series. If you take the time to explore that and want to share your findings, we'd love to hear about them though!

I'm going to convert this to a discussion since we won't be making changes to the repo code, but thanks for bringing this up!
